pkg/cloud/azure/actuators/machineset/controller_suite_test.go

```diff
@@ -26,19 +26,24 @@ import (
 	. "github.com/onsi/gomega"
 	configv1 "github.com/openshift/api/config/v1"
 	machinev1 "github.com/openshift/api/machine/v1beta1"
+	"github.com/openshift/machine-api-provider-azure/pkg/cloud/azure/actuators/machine"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
 const (
-	timeout = 10 * time.Second
+	timeout                  = 10 * time.Second
+	globalInfrastructureName = "cluster"
 )
 
 var (
 	cfg     *rest.Config
 	testEnv *envtest.Environment
+	clt     client.Client
```

Comment:
Nit, we have this same pattern across a lot of our repos but call this variable `k8sClient`, would you mind renaming it for consistency?

Reply:
I will make the change.